osm2streets/src/transform/intersection_geometry.rs

```diff
@@ -4,12 +4,6 @@ use geom::{Circle, Distance};
 use crate::{IntersectionControl, IntersectionKind, StreetNetwork};
 
 pub fn generate(streets: &mut StreetNetwork, timer: &mut Timer) {
-    // Initialize trimmed_center_line to the corrected center
```

Comment:
I'm still puzzling through how it happens, but take a look at the diff in `aurora_sausage_link`. It looks like we lose clipping to the map edge. In `Road::new` we're immediately doing `update_center_line`, so this should be set...
OHHH no it's simple. At the end of `streets_reader/src/clip.rs`, we need to call `update_center_line`. #127 would make this simpler possibly

Reply:
The approach in this PR is to treat `center_line` as derived state and update it as soon as its invalidated (like `kind`). 4b03cdc actually follows through on that and adds the required `update_center_line` calls (after the initial one in `Road::new`), which fixes the `clip` problem, and the one in my original screenshot.